src/services/SolidService.ts

```diff
@@ -30,6 +31,22 @@ class SolidService {
     this.doneCreatingFiles = false;
   }
 
+  uploadFile = async (
+    file: any,
+    targetLink: string,
+    session: any,
+  ): Promise<any> => {
```

Comment:
Promise<any>, why using any when not returning anything? I think `Promise<void>` is correct here.